src/store/actions.ts

```diff
@@ -312,37 +312,47 @@ export const fetchCorpNum = async ({ getters }, corpNum: string): Promise<any> =
 }
 
 // FUTURE: not an action - move it to another module?
-export const checkCOLIN = ({ getters }, corpNum: string) => {
-  const myToken = sessionStorage.getItem('KEYCLOAK_TOKEN')
-
-  // check entity for the corp num first
-  let url = `${appBaseURL}/businesses/${corpNum}`
-  return axios.get(url, {}).catch(error => {
-    if (error.response && error.response.status === 404) {
-      // Remove BC prefix as Colin only supports base number with no prefix for BC's
-      const cleanedCorpNum = corpNum.replace(/^BC+/i, '')
-      url = `${appBaseURL}/colin/${cleanedCorpNum}`
-      return axios.post(url, {}).then(response => {
-        if (response.data.directors === 'Not Available' && response.data.incorporated === 'Not Available') {
-          const error = new Error('Not Found')
-          return Promise.reject(error)
+async function checkCOLIN ({ getters }, corpNum: string): Promise<BusinessFetchIF> {
```

Comment:
Note that this function is now in "await" format (instead of "then" format), which I think is cleaner (and is also more modern).